packages/block-editor/src/components/block-list/index.js

```diff
@@ -7,45 +7,61 @@ import classnames from 'classnames';
  * WordPress dependencies
  */
 import { AsyncModeProvider, useSelect } from '@wordpress/data';
-import { useRef, forwardRef } from '@wordpress/element';
+import {
+	useRef,
+	forwardRef,
+	createContext,
+	useState,
+} from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import BlockListBlock from './block';
 import BlockListAppender from '../block-list-appender';
-import RootContainer from './root-container';
 import useBlockDropZone from '../use-block-drop-zone';
+import useInsertionPoint from './insertion-point';
+import BlockPopover from './block-popover';
 
 /**
  * If the block count exceeds the threshold, we disable the reordering animation
  * to avoid laginess.
  */
 const BLOCK_ANIMATION_THRESHOLD = 200;
 
+export const BlockNodes = createContext();
+export const SetBlockNodes = createContext();
+
 function BlockList(
 	{ className, placeholder, rootClientId, renderAppender },
 	ref
 ) {
-	const Container = rootClientId ? 'div' : RootContainer;
```

Comment:
`rootClientId` will always be falsy since `InnerBlocks doesn't use this component anymore.

Reply:
When did InnerBlocks stop using this component?